templates/cukinia-observer.conf.j2

```diff
@@ -0,0 +1,15 @@
+# Copyright (C) 2020, RTE (http://www.rte-france.com)
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
I will move this file inside the cukinia-tests submodule.

Reply:
Ack, see the PR on cukinia-tests repo